services/QuillLMS/client/app/bundles/Teacher/helpers/userSubscription.ts

```diff
@@ -0,0 +1,10 @@
+interface User {
```

Comment:
Just another preference, so take or leave, but I'd put this helper file in the `components/classrooms` folder rather than at the top-level -- we can always hoist it later if we want to expand its use, because I'm sure we use similar functions elsewhere, but I just think it's tidier to keep helpers closer to where they're applied unless we know we want to include their function in multiple folders.